### PR TITLE
Add swap functionality

### PR DESCRIPTION
**Pull request by Nukesor**
Hey,

I added a `swap` flag to swap the currently active window with the selected window.

It's quite handy for multi-monitor setups :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - ReleaseDate
 
+## [1.5.0] - unreleased
```
Comment (svenstaro) [inline CHANGELOG.md:12]: Put this under `[Unreleased]` or my release script won't work. :{

**Review by svenstaro — CHANGES_REQUESTED** [review]
Thanks, super neat! Just tiny nits.

**Comment by svenstaro** [comment]
Also please fix the lints. )

**Comment by svenstaro** [comment]
Top nice! 👍

**Comment by uunnxx** [comment]
It's awesome!
Thank you <3

**Pull request merged**